gtm/GTM Container for Gluteboost.json

```diff
@@ -0,0 +1,7894 @@
+                "parameter": [
+                    {
+                        "type": "INTEGER",
+                        "key": "dataLayerVersion",
+                        "value": "2"
+                    },
+                    {
+                        "type": "BOOLEAN",
+                        "key": "setDefaultValue",
+                        "value": "false"
+                    },
+                    {
+                        "type": "TEMPLATE",
+                        "key": "name",
+                        "value": "ecommerce.detail.products.0.category"
+                    }
+                ],
+                "fingerprint": "1676908246720",
+                "parentFolderId": "3",
+                "formatValue": {
+                    "caseConversionType": "LOWERCASE"
+                }
+            },
+            {
+                "accountId": "6073593102",
+                "containerId": "102923929",
+                "variableId": "86",
+                "name": "DLV - Customer First Name",
+                "type": "v",
+                "parameter": [
+                    {
+                        "type": "INTEGER",
+                        "key": "dataLayerVersion",
+                        "value": "2"
+                    },
+                    {
+                        "type": "BOOLEAN",
+                        "key": "setDefaultValue",
+                        "value": "false"
+                    },
+                    {
+                        "type": "TEMPLATE",
+                        "key": "name",
+                        "value": "customerFirstName"
+                    }
+                ],
+                "fingerprint": "1676908339115",
+                "parentFolderId": "3",
+                "formatValue": {}
+            },
+            {
+                "accountId": "6073593102",
+                "containerId": "102923929",
+                "variableId": "88",
+                "name": "JS - View Item List - Items",
+                "type": "jsm",
+                "parameter": [
+                    {
+                        "type": "TEMPLATE",
+                        "key": "javascript",
+                        "value": "function() {\n    var gtmproductArray = {{DLV - Ecommerce - Impressions - Products}};\n    var gaproductArray = [];\n    for (var i=0; i<gtmproductArray.length; i++){\n        var obj = {};\n        obj.id = gtmproductArray[i].productId;\n        gaproductArray.push(obj);\n    }\n    return gaproductArray;\n    }"
+                    }
+                ],
+                "fingerprint": "1676908339115",
+                "parentFolderId": "3",
+                "formatValue": {}
+            },
+            {
+                "accountId": "6073593102",
+                "containerId": "102923929",
+                "variableId": "91",
+                "name": "JS - Ecommerce - Add - Items - Meta",
+                "type": "jsm",
+                "parameter": [
+           
```

Comment:
@ankitnagarsheth the merchant has two Facebook pixels? Is this valid?

Reply:
Thats right! They have 2 facebook pixels.